### PR TITLE
feat: add OF tracking docs for Node + Web

### PR DESCRIPTION
**Pull request by jonathannorris**

### CODE REVIEW

File: docs/sdk/client-side-sdks/javascript/javascript-openfeature.md [hunk 1]
```diff
@@ -23,15 +23,18 @@ DevCycle provides a Javascript implementation of the OpenFeature Web Provider in
 
 Install the OpenFeature Web SDK and DevCycle Web Provider:
 
-[//]: # (wizard-install-start)
+[//]: # 'wizard-install-start'
```
Comment (jonathannorris) [inline docs/sdk/client-side-sdks/javascript/javascript-openfeature.md:26]: hum... these were auto-formatted like this

Comment (JamieSinn) [reply docs/sdk/client-side-sdks/javascript/javascript-openfeature.md:26]: prettier has opinions. I don't question them

**Review by JamieSinn — APPROVED** [review]

File: docs/sdk/client-side-sdks/javascript/javascript-openfeature.md (hunk 1 above)
Comment (jonathannorris) [reply docs/sdk/client-side-sdks/javascript/javascript-openfeature.md:26]: yea I just wonder if it will work with the on-boarding stuff that uses this.

Comment (JamieSinn) [reply docs/sdk/client-side-sdks/javascript/javascript-openfeature.md:26]: onboarding uses the labs sdks - no the prod ones

Comment (JamieSinn) [reply docs/sdk/client-side-sdks/javascript/javascript-openfeature.md:26]: oh I see what you mean - one way to find out?

Comment (jonathannorris) [reply docs/sdk/client-side-sdks/javascript/javascript-openfeature.md:26]: looks like these need to be reverted: 
image

Comment (JamieSinn) [reply docs/sdk/client-side-sdks/javascript/javascript-openfeature.md:26]: yup

**Pull request merged**